input/_partials/_head.cshtml

```diff
@@ -27,7 +27,7 @@
   <link href="https://fonts.googleapis.com/css?family=Bree+Serif" rel="stylesheet" data-no-mirror=data-no-mirror />
   <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet" data-no-mirror=data-no-mirror />
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" data-no-mirror=data-no-mirror />
-  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous" data-no-mirror=data-no-mirror />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" crossorigin="anonymous" data-no-mirror=data-no-mirror />
```

Comment:
Needed to bump the version of Font Awesome to get the Dev icon. Couldn't find an `integrity` value (they're heavily pushing custom bundle files now) so just left it off.